src/tests/test_plot.py

```diff
@@ -133,5 +162,5 @@ def test_internal_histogram_exception(tmp_empty, ip, cell, error_type, error_mes
 """
     )
     out = ip.run_cell(cell)
-    assert isinstance(out.error_in_exec, error_type)
-    assert str(error_message).lower() in str(out.error_in_exec).lower()
+    assert out.error_in_exec is error_check
```

Comment:
No need of checking error=None if there's no error. Please remove it. Define cell in this internal function and remove the parametrize.